content/it/docs/concepts/overview/kubernetes-api.md

```diff
@@ -0,0 +1,126 @@
+
+**Esempi per ottenere le specifiche OpenAPI**:
+
+Prima della 1.10 | Dalla versione 1.10 di Kubernetes
+---------------- | -----------------------------
+GET /swagger.json | GET /openapi/v2 **Accept**: application/json
+GET /swagger-2.0.0.pb-v1 | GET /openapi/v2 **Accept**: application/com.github.proto-openapi.spec.v2@v1.0+protobuf
+GET /swagger-2.0.0.pb-v1.gz | GET /openapi/v2 **Accept**: application/com.github.proto-openapi.spec.v2@v1.0+protobuf **Accept-Encoding**: gzip
+
+Kubernetes implementa per le sue API anche una serializzazione alternativa basata sul formato Protobuf che è stato pensato principalmente per la comunicazione intra-cluster, documentato nella seguente [proposta di design](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/protobuf.md) ed i files IDL per ciascun schema si trovano nei *Go packages* che definisco i tipi delle API.
+
+Prima della versione 1.14, l'*apiserver* di Kubernetes espone anche un'*endpoint*, `/swaggerapi`, che può essere usato per ottenere
+le documentazione per le API di Kubernetes secondo le specifiche [Swagger v1.2](http://swagger.io/) .
+Questo *endpoint* è deprecato, ed è stato rimosso nella versione 1.14 di Kubernetes.
+
+## Versionamento delle API
+
+Per facilitare l'eliminazione di campi specifici o la modifica della rappresentazione di una data risorsa, Kubernetes supporta molteplici versioni della stessa API disponibili attraverso differenti percorsi, come ad esempio `/api/v1` oppure
+`/apis/extensions/v1beta1`.
+
+Abbiamo deciso di versionare a livello di API piuttosto che a livello di risorsa o di campo per assicurare che una data API rappresenti una chiara, consistente vista delle risorse di sistema e dei sui comportamenti, e per abilitare un controllo di accesso sia per le API in via di decommissionamento che per quelle sperimentali.
+
+Si noti che il versionamento delle API ed il versionamento del Software sono indirettamente collegati. La [proposta per il versionamento delle API e dei rilasci del Sotware](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md) descrive la relazione tra il versionamento delle API ed il versionamento del Software.
+
+Differenti versioni delle API implicano differenti livelli di stabilità e supporto.  I criteri per ciascuno livello sono descritti in dettaglio nella [documentazione delle modifiche alle API](https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions).  Queste modifiche sono qui ricapitolate:
+
+- Livello alpha:
+  - Il nome di versione contiene `alpha` (e.g. `v1alpha1`).
+  - Potrebbe contenere dei *bug*.  Abiliare questa funzionalità potrebbe esporre al rischio di *bugs*.  Disabilitata di default.
+  - Il supporto di questa funzionalità potrebbe essere rimosso in ogni momento senza previa notifica.
+  - Questa API potrebbe non essere più compatibile in rilasci futuri del Software senza previa notifica.
```

Comment:
```suggestion
  - Questa API potrebbe cambiare in mod incompatibile in rilasci futuri del Software e senza previa notifica.
```

Reply:
typo in the suggestion I'll fix it manually later